### PR TITLE
Signup page footer

### LINKED ISSUE #310: Repcoin about and contact should be available on signup page
You shouldn't have to sign up to read what Repcoin is or who made it

### PR DESCRIPTION
**Pull request by ritmatter**
Fixes #310

### CODE REVIEW

File: js/auth.jsx [hunk 1]
```diff
@@ -36,35 +35,38 @@ var auth = {
   },
 
   getCurrentUser: function(cb) {
-    function getCurrentUserLocal(cb) {
+    // Attempt to get the current user locally and check the server if that fails
+    var currentUser = getCurrentUserLocal();
```
Comment (ritmatter) [inline js/auth.jsx]: Attempt to get the user locally, and try remote of that fails

File: js/auth.jsx [hunk 2]
```diff
@@ -36,35 +35,38 @@ var auth = {
...
-        return false;
-      } else {
-        cb(currentUser);
-        return true;
+        return null;
       }
+      return JSON.parse(window.localStorage.getItem('currentUser'));
     }
 
     function getCurrentUserRemote(cb) {
```
Comment (ritmatter) [inline js/auth.jsx]: Make sure to do the callBack on null if no user is logged in. JSON will send a null user back as "", which caused problems.

File: js/auth.jsx [hunk 3]
```diff
@@ -36,35 +35,38 @@ var auth = {
...
+          if (user) {
+            window.localStorage.setItem('currentUser', JSON.stringify(user));
+            cb(user);
+          } else {
+            cb(null);
+          }
         },
         error: function(xhr, status, err) {
-          console.error(this.props.userId, status, err.toString());
+          console.error(xhr.responseText);
```
Comment (krazemon) [inline js/auth.jsx]: There are a probably a bunch of places we should change this. I used it in one of my recent components but haven't gone back and fixed it on the others.

Comment (krazemon) [inline js/auth.jsx]: Anyway, irrelevant to this PR.

**Comment by krazemon** [comment]
:+1:

**Pull request merged**